volume/store/store.go

```diff
@@ -309,16 +309,20 @@ func (s *VolumeStore) create(name, driverName string, opts, labels map[string]st
 // This makes sure there are no races between checking for the existence of a volume and adding a reference for it
 func (s *VolumeStore) GetWithRef(name, driverName, ref string) (volume.Volume, error) {
 	name = normaliseVolumeName(name)
```

Comment:
Make sense. Maybe next PR? This time, for restoring containers at the daemon start, only needs these 2 places.